### ISSUE TITLE
can't select Methods in "Jade Method Browser"

### ISSUE THREAD

**Issue by PeterMoe** [jgfoster/Jade.bak#214]
Hi James, 

can't select Method in Implementors and Senders Window

```
System class>>#signal:args:signalDictionary: @8 line 15
String (Object)>>#doesNotUnderstand: @6 line 14
String (Object)>>#_doesNotUnderstand: @1 line 6
JadeServer64bit24 (JadeServer)>>#describeMethodNamed:inClass: @2 line 5

```

GS246 dosen't  have the method 
`decodeToUnicode`
and GS24x can't use
`compiledMethodAt:otherwise:` must use `compiledMethodAt:ifAbsent:`

please give me the "Old method" back and please overwrite for GS3x in subclass under JadeServer64bit24

This was my "quick fix" on class JadeServer64bit24 i don't want to do too much changes, because i don't understand the GitHub "stuff".
I know you can do it much better.

```
describeMethodNamed: aUtf8 inClass: aClass
	"Provide info needed to create a GsMethod in Jade client"
	"
	2019-04-15 PeterMoe:
		overwritten because method decodeToUnicode in GS24x not implemented. need respondsTo: for compatibility for subclasses.
		changed from compiledMethodAt:otherwise: to compiledMethodAt:ifAbsent: because in GS24x not implemented.
	"
	| aMethod |
	(aUtf8 respondsTo: #decodeToUnicode) ifTrue: [^super describeMethodNamed: aUtf8 inClass: aClass].
	(aMethod := aClass compiledMethodAt: aUtf8 ifAbsent: [nil]) isNil ifTrue: [^'']. 
	writeStream := self _writeStreamOnString.
	self _describeMethod: (self homeMethodFor: aMethod).
	^self _encodeString: writeStream contents
```

I love Jade and want to use the "Last Version", please be careful for backward compatibility to GS24x.

Regards
Peter

**Comment by thelliez** [comment]
Same problem here with a 2.4 repository


Thierry

**Comment by thelliez** [comment]
How do you reverse back?  I reinstalled 2.3.7 which I think is the version
I had working before.  But I am seeing the same issue with 2.3.9 ,2.3.8 and
2.3.7.


Thanks,
Thierry

**Comment by PeterMoe** [comment]
Hello James,
in Jade v2.3.11 it works. I think you have fixed it with Commit: f1c19bc Better handling of non-zero method environments.

Regards,
Peter
